docs/branding.stories.js

```diff
@@ -1,4 +1,14 @@
-export const ensemble_demo = `
+import { storiesOf } from '@open-wc/demoing-storybook';
+import '../src/index';
+import '../assets/css/shared.css';
+
+storiesOf('branding', module)
+  // .addDecorator(withKnobs)
+  .add('kitchen sink', () => header_demo + ensemble_demo)
+  .add('bulib-header', () => header_demo)
```

Comment:
header's been added and the kitchen sink isn't all that important. let's ditch `docs/` entirely